### PR TITLE
Run 25

### PR DESCRIPTION
**Pull request by kayoumido**

### CODE REVIEW

**Comment by kayoumido** [comment]
Working on fixing conflicts

**Comment by kayoumido** [comment]
Solved conflicts can be merged

File: src/pages/filters/filters.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+.modal__item {
+  text-align: center;
+  border: none !important;
+  font-size: 25px;
+  background: none;
+  a {
+    color: #fff;
+  }
+  .item-inner {
```
Comment (Westixy) [inline src/pages/filters/filters.scss]: Avoid important flag if you can

Comment (kayoumido) [reply src/pages/filters/filters.scss]: I know but in this case not possible to avoid it :/

**Pull request merged**